api/events/schema.py

```diff
@@ -35,6 +39,80 @@ class Meta:
         model = EventsModel
 
 
+class BookEvent(graphene.Mutation):
+    """
+        Book Calendar events
+    """
+    response = graphene.String()
+
+    class Arguments:
+        event_title = graphene.String(required=True)
+        start_date = graphene.String(required=True)
+        start_time = graphene.String(required=True)
+        duration = graphene.Float(required=False)
+        attendees = graphene.String(required=False)
+        organizer = graphene.String(required=False)
+        description = graphene.String(required=False)
+        room = graphene.String(required=False)
```

Comment:
I think the room should be required since you are booking a room

Reply:
Thank you @mifeille, I'll make the change and revert back to you.